desktop/src/roles/role-form/role-form.component.scss

```
@@ -0,0 +1,579 @@
+@use "../../variables.scss" as variables;
+
+// ----- Token map (design --rw-* -> variables.scss) -----
+$primary-50: map.get(variables.$primary-palette, 50);
+$primary-100: map.get(variables.$primary-palette, 100);
+$primary-300: map.get(variables.$primary-palette, 300);
+$primary-400: map.get(variables.$primary-palette, 400);
+$primary-500: map.get(variables.$primary-palette, 500);
+$primary-600: map.get(variables.$primary-palette, 600);
+$primary-700: map.get(variables.$primary-palette, 700);
+
+$slate-50: map.get(variables.$accent-palette, 50);
```

Stylelint (17.12.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)